Makefile

```diff
@@ -71,7 +71,7 @@ licenseheaders: build/licenseheaders.${CONDA_ENV_NAME}.build-stamp
 
 build/licenseheaders.${CONDA_ENV_NAME}.build-stamp: $(PYTHON_FILES) doc/mit-license.tmpl
 	# Note: to avoid makefile dependency loops, we don't touch the setup.py files as that would force the conda-env to be rebuilt.
-	conda run -n ${CONDA_ENV_NAME} licenseheaders -t doc/mit-license.tmpl -E .py .sh .ps1 -x mlos_bench/setup.py mlos_core/setup.py
+	conda run -n ${CONDA_ENV_NAME} licenseheaders -t doc/mit-license.tmpl -E .py .sh .ps1 .sql -x mlos_bench/setup.py mlos_core/setup.py
```

Comment:
Adding this to make sure future schemas have the license header (you already included it, but figured we should have it automated too).